docker-signal.go

```diff
@@ -49,17 +50,25 @@ func main() {
 	var (
 		pid      int
 		debugger bool
+		legacy   bool
 	)
 	flag.BoolVar(&debugger, "debugger", false, "Signal a debugger event rather than stackdump.")
+	flag.BoolVar(&legacy, "legacy", false, "Trigger a stackdump for 18.09 and below docker engine")
```

Comment:
wondering if instead we should add back the flag that was added in https://github.com/moby/docker-signal/pull/1 (with the typos fixed 😅)


Note that I see that older versions were tagged, so alternatively it would be possible to just download v0.1 or v0.2 for situations where an older daemon is used, in which case no changes would be needed for the current version

Reply:
I had discussed the option of >1 versions of the binary with @ameyag prior to this change being created.  I was trying to think of a better way to handle this, and at the time the command-line option looked to be the path of least resistance.  Another option would be to give the binaries unique filenames that reflect the versions they support, but this feels awkward.

Also note that [Microsoft has a troubleshooting page which links to this](https://docs.microsoft.com/en-us/virtualization/windowscontainers/troubleshooting#obtaining-stack-dump).  Even with submitting a doc issue or PR, we don't have a say over how quickly the page gets updated to reflect the change in this repo.

Another way of phrasing this: what's the best way to signal a user to let them know that they've downloaded the wrong binary?